glossary.rst

```diff
@@ -0,0 +1,46 @@
+.. _glossary:
+
+========
+Glossary
+========
+
+This page contains definitions for Flux-specific terms and concepts mentioned
+throughout the various guides and docs on our sites.
+
+------------------------------------------------------------------------------
+
+**Flux Broker**
+  A distributed message broker daemon that provides communications services
+  within a Flux instance.
+
+**Flux Instance**
+  A group of flux-broker processes which are launched via any parallel launch
+  utility that supports PMI (Process Manager Interface).
+
+**Hostlist**
+  A format supported by existing HPC tools to create a compact list of hosts by
+  name, consisting of a common prefix, a number, and an optional suffix.
+
+**Jobspec**
```

Comment:
Definitely need an example here, even if simple! And then to say what the Jobspec specifically does. E.g., what is creating it? If it's a program, is it being launched somewhere?

Reply:
Maybe some of the terms and definitions listed here could be just slotted in as a first pass?

https://flux-framework.readthedocs.io/en/latest/guides/admin-guide.html#software-components